search/services/index.py

```diff
@@ -159,27 +159,45 @@ def _group_terms(query: Query) -> tuple:
         return terms[0]
 
     @staticmethod
-    def _field_term_to_q(term) -> Q:
-        if term.field in ['title', 'abstract']:
+    def _field_term_to_q(field: str, term: str) -> Q:
```

Comment:
This is now used by both advanced and simple search, since we're doing precisely the same thing with each field.